recipe/meta.yaml

```diff
@@ -7,12 +7,12 @@ package:
 source:
     fn: Shapely-{{ version }}.tar.gz
     url: https://github.com/Toblerity/Shapely/archive/{{ version }}.tar.gz
-    sha256: 3a0940dab0bc0de0d572b7993f61a4d9a38ce17608573e676c1413b5de0cf776
+    sha256: 31fee47d9208078a19f40a451c0c99c085d16343e66cbd0dd5af0af6f48cfc3a
```

Comment:
Weird. I computed `85dd42a3e229f434642ad39744ebc82cd352931c6686d858bd9607649b5a4d8a`

Did you use PyPI's tarball instead of the git-archive for the checksum?